server/certstore_windows_test.go

```diff
@@ -59,18 +59,18 @@ func runConfiguredLeaf(t *testing.T, hubPort int, certStore string, matchBy stri
 						cert_store: "%s"
 						cert_match_by: "%s"
 						cert_match: "%s"
+						ca_certs_match: %s
 
 						# Above should be equivalent to:
 						# cert_file: "../test/configs/certs/tlsauth/client.pem"
 						# key_file: "../test/configs/certs/tlsauth/client-key.pem"
-
-						ca_file: "../test/configs/certs/tlsauth/ca.pem"
+						# ca_file: "../test/configs/certs/tlsauth/ca.pem"
```

Comment:
Should this be deleted if commented out?

Reply:
It's actually equivalent now with the CA search field above, so I moved it up a few lines.  I can add a comment.